<launch>

<arg name="path_data_logging" default = "/mnt/DATA/tmp/ClosedNav/"/>

<!-- initialize controller -->
<node name="turtlebot_controller" pkg="turtlebot_trajectory_controller" type="trajectory_controller_node" output="screen" args="" >
    <remap from="odom" to="visual/odom"/>
</node>

<!-- initialize rosbag recording -->
<node name="data_logging" type="record" pkg="rosbag" args="/visual/odom /odom_sparse /ground_truth/state /ORB_SLAM/camera_pose_in_imu /desired_path /turtlebot_controller/trajectory_controller/desired_trajectory -O $(arg path_data_logging).bag">
</node>

</launch>
